spec/views/users/passwords/edit.html.erb_spec.rb

```diff
@@ -68,10 +68,12 @@
       expect(rendered).to_not have_content(t('links.cancel'))
     end
 
-    it 'aria described by is blank' do
```

Comment:
I think we should just remove this `it` test altogether, since we're not expecting the `aria-describedby` behavior to be affected by `require_password_change?`.